v2/cf-nodejs-logging-support/src/lib/middleware/utils.ts

```diff
@@ -0,0 +1,21 @@
+import Config from "../config/config";
+import ConnectService from "./framework-services/connect";
+import ExpressService from "./framework-services/express";
+import HttpService from "./framework-services/plainhttp";
+import RestifyService from "./framework-services/restify";
+import { IFrameworkService } from "./interfaces";
+
+export function assignFrameworkService(): IFrameworkService {
+    const framework = Config.getInstance().getFramework();
+    switch (framework) {
+        //insert your custom framework logger here
```

Comment:
Please remove/update this comment
- "custom" frameworks are not supported (yet).
- "framework logger" -> "framework service"